### PR TITLE
PMM-797 remove Host from redirect location

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: configurator/config/config.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ func ParseConfig() (c PMMConfig) {
 	flag.Parse() // command line should overide config
 	c.setDefaultValues()
 	c.setLogger()
+	c.validateValues()
```
Comment (AlekSi) [inline configurator/config/config.go]: I would say that validation should validate values and exit with errors, not silently fix them.

File: configurator/config/config.go [hunk 2]
```diff
@@ -131,3 +132,9 @@ func (c *PMMConfig) setLogger() {
 		log.SetOutput(io.MultiWriter(logFile, os.Stderr))
 	}
 }
+
+func (c *PMMConfig) validateValues() {
+	if len(c.PathPrefix) > 0 && string(c.PathPrefix[0]) != "/" {
```
Comment (AlekSi) [inline configurator/config/config.go]: `strings.HasPrefix()`.

Comment (delgod) [inline configurator/config/config.go]: fixed

File: configurator/config/config.go (hunk 1 above)
Comment (delgod) [inline configurator/config/config.go]: fail forward in action :)

Comment (delgod) [inline configurator/config/config.go]: warning is added

**Pull request merged**